### PR TITLE
Script addition : GitHub good first issues

### LINKED ISSUE #1006: Scrap Good First Issues from GitHub Repositories
# Description

Create a Script to scrap Good First Issues from GitHub Repositories. We can have the GitHub repositories in the following ways: 

- Having a JSON File where all the repositories are stored.
- Scrapping from the top repositories that are trending on GitHub Explore 
- Scrapping from the repos that we/or someone else has starred 

You can have better ways of taking the inputs, but these three are the most important ones, that we are looking for.

## Language

- [X] Golang
- [X] Javascript
- [X] Python

### PR DESCRIPTION
**Pull request by jhamadhav**
# Description

 Script to scrap Good First Issues from GitHub Repositories

Fixes #1006 

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)

# Checklist:

- [x] My code follows the style guidelines(Clean Code) of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have created a helpful and easy to understand `README.md`
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests/screenshots(if any) that prove my fix is effective or that my feature works

## Output
Output image

### CODE REVIEW

**Comment by jhamadhav** [comment]
@vybhav72954 @HarshCasper Please review the PR.

File: JavaScript/Github_Good_First_Issues/DB.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+let repoDB = [
+    {
+        "owner": "HarshCasper",
+        "name": "Rotten-Scripts"
+
```
Comment (HarshCasper) [inline JavaScript/Github_Good_First_Issues/DB.js:5]: Can you add another project to this: https://github.com/TesseractCoding/NeoAlgo 

Just as an example :wink:

File: package.json [hunk 2]
```diff
@@ -0,0 +1,5 @@
+{
+  "dependencies": {
+    "node-html-parser": "^3.1.5"
+  }
+}
```
Comment (HarshCasper) [inline package.json:5]: Why do we need this here?

File: package-lock.json [hunk 3]
```diff
@@ -0,0 +1,214 @@
+{
```
Comment (HarshCasper) [inline package-lock.json:1]: Why do we need a `package-lock.json` on the root?

File: JavaScript/Github_Good_First_Issues/README.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+- Get your [`Github Developer Token`](https://dev.to/gr2m/github-api-authentication-personal-access-tokens-53kd) and follow the steps there to generate your **Token key**
+- In a file named `token.js` paste the token.
+- Open terminal and do the following : 
+- `cd Rotten-Scripts\JavaScript\Github_Good_First_Issues`
+- Run `npm install` to install all necessary dependencies
+- Run `node Github_Good_First_Issues.js` and Voila! you are ready to go 😉
+
+## Output
+
+![Output Pic](https://i.imgur.com/Bcfvpu4.png)
```
Comment (HarshCasper) [inline JavaScript/Github_Good_First_Issues/README.md:25]: Can you add examples for rest as well?

**Review by HarshCasper — CHANGES_REQUESTED** [review]

File: JavaScript/Github_Good_First_Issues/DB.js (hunk 1 above)
Comment (jhamadhav) [reply JavaScript/Github_Good_First_Issues/DB.js:5]: Sure 😁

File: package.json (hunk 2 above)
Comment (jhamadhav) [reply package.json:5]: Because I'm scrapping the trending section so I need that to parse the html

File: package-lock.json (hunk 3 above)
Comment (jhamadhav) [reply package-lock.json:1]: Oh I will remove that Asap

File: JavaScript/Github_Good_First_Issues/README.md (hunk 4 above)
Comment (jhamadhav) [reply JavaScript/Github_Good_First_Issues/README.md:25]: Sure 👍

File: package.json (hunk 2 above)
Comment (HarshCasper) [reply package.json:5]: I mean, why on the root? We keep root clean from these files.

Comment (jhamadhav) [reply package.json:5]: 😬seems like a blunder from my side will fix it asap

**Comment by jhamadhav** [comment]
@HarshCasper I have made the requested changes please check

**Review by HarshCasper — APPROVED** [review]

**Pull request merged**